### PR TITLE
Enable preferreduilang for fsi

### PR DESCRIPTION
**Pull request by KevinRansom**
Enable the preferreduilang switch for fsi.exe.

````
fsi.exe --preferreduilang:de-DE

Microsoft (R) F# Interactive, Version 4.1
Copyright (c) Microsoft Corporation. Alle Rechte vorbehalten.

Um Hilfe zu erhalten, geben Sie "#help;;" ein.

> #quit;;
````

### CODE REVIEW

**Comment by forki** [comment]
hugs!

**Comment by cartermp** [comment]
Docs update: https://github.com/dotnet/docs/pull/4193

**Comment by cartermp** [comment]
@dotnet-bot test this please

**Comment by realvictorprm** [comment]
Great!


Am 24.01.2018 9:27 vorm. schrieb "Kevin Ransom (msft)" <
notifications@github.com>:

File: src/fsharp/fsi/fsi.fs [hunk 1]
```diff
@@ -2494,19 +2494,17 @@ type FsiEvaluationSession (fsi: FsiEvaluationSessionHostConfig, argv:string[], i
...
-      | Some _ -> ()
-      | None -> tcConfigB.lcid <- lcidFromCodePage
-
-    // Set the ui culture
-    do 
-      match fsiOptions.FsiLCID with
-      | Some(n) -> Thread.CurrentThread.CurrentUICulture <- new CultureInfo(n)
-      | None -> ()
+#if FX_LCIDFROMCODEPAGE
+      ignore lcidFromCodePage
```
Comment (brettfo) [inline src/fsharp/fsi/fsi.fs]: Does this do anything?

Comment (KevinRansom) [reply src/fsharp/fsi/fsi.fs]: I don't think so ... I will check.

**Pull request merged**